ReactCommon/jsi/V8Runtime_shared.cpp

```diff
@@ -239,6 +239,12 @@ namespace facebook { namespace v8runtime {
     }
   }
 
+  void  V8Runtime::Log(const std::string& message, const unsigned int logLevel) {
+    if (logger_) {
+      (*logger_)("V8Runtime:: " + message, logLevel);
+    }
```

Comment:
This can be inlined .. just keep the definition along with declaration ..

Reply:
Done.